frontend/src/metabase/collections/components/CollectionHeader/CollectionCaption.tsx

```diff
@@ -74,5 +82,20 @@ const CollectionCaption = ({
   );
 };
 
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export default CollectionCaption;
+const CollectionCaptionIcon = ({ collection }: { collection: Collection }) => {
+  // we only show icons for "special" collections with types
+  if (!collection.type) {
+    return null;
+  }
+
+  const icon = getCollectionIcon(collection);
+  const tooltip = getCollectionTooltip(collection);
+
+  if (!icon) {
+    return null;
+  }
+
+  return (
+    <Icon size={24} name={icon.name} color={color("brand")} tooltip={tooltip} />
+  );
+};
```

Comment:
I'm curious whether this should go in the enterprise directory?

Currently, there's only one collection type with a special icon, and it's for an enterprise only feature. I wouldn't say the icon itself is an enterprise feature. The actual feature is easy access to the audit tables and the creation of premade questions + dashboards, all of which happens on the backend.

Reply:
I'm a little inclined to say that it should, given that that's what we do for official collections. I also just find it a little strange that we have 2 different components next to each show an icon if conditions are met

Reply:
I combined both icons into a single component. I think this might end up being a little different when we move some of this logic to a plugin in https://github.com/metabase/metabase/issues/31212